### PR TITLE
chore: add badges dark/light pictures and more documentation to README.md

### PR DESCRIPTION
**Pull request by polaroi8d**
PR contains the following updates: 
- Add dark & light pictures to support both appearances
- Remove unnecessary `package-lock.json` from the root folder
- Extend the README with how it works and use cases sections

### CODE REVIEW

**Review by robot9706 — APPROVED** [review]
One comment, otherwhise LGTM 😎

File: README.md [hunk 1]
```diff
@@ -1,62 +1,111 @@
...
-# dyrector.io - The open source internal delivery platform
+## Overview
 
-[![License](https://img.shields.io/badge/licence-Apache%202.0-brightgreen.svg?style=flat)](LICENSE)
-[![Discord Budget](https://img.shields.io/discord/797082431902449694)](https://discord.gg/pZWbd4fxga)
-[![codecov](https://img.shields.io/codecov/c/github/dyrector-io/dyrectorio/develop?logo=Go&token=F5TZTAJTKX)](https://codecov.io/gh/dyrector-io/dyrectorio)
+Life isn’t about application management. Why should your team focus so much of its precious time and effort on it when things can be simpler?
 
-## Overview
+dyrector.io is for you if you’d like to shift your team’s focus from repeatable steps of multi-instance deployments to delivering value to your users. And when a deployment is successful or any error occurs, the team gets notified on chat so they can react as soon as possible.
```
Comment (balanceee) [inline README.md:27]: I know there is no grammatical rule for starting a sentence with an "and" but can you please change it to "also" or something similiar . Its kinda weird to my eyes

**Review by balanceee — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (polaroi8d) [reply README.md:27]: Good point I'll fix it. Thanks!

**Review by robot9706 — APPROVED** [review]
LGTM 😎

File: README.md (hunk 1 above)
Comment (polaroi8d) [reply README.md]: Updated could you please check it?

**Review by balanceee — APPROVED** [review]
+1

**Pull request merged**